fem/coefficient.hpp

```diff
@@ -848,6 +1019,12 @@ class MatVecCoefficient : public VectorCoefficient
 public:
    MatVecCoefficient(MatrixCoefficient &A, VectorCoefficient &B);
```

Comment:
Somehow related suggestion: Could we rename this class so it matches all the other compound coefficients? For instance `MatrixVectorCoefficient` or `MatrixVectorProductCoeffcient`?

Reply:
This sounds fine to me.  Perhaps we can add a `typedef` for backward compatibility since `MatVec` is such a common abbreviation.  If there are no objections I'll do this.

Reply:
I haven't heard any objections so I made the change.  Thanks, @aschaf !